src/assert.js

```diff
@@ -57,8 +57,10 @@ class Assert {
 	// Exports test.push() to the user API
 	// Alias of pushResult.
 	push( result, actual, expected, message, negative ) {
-		console.warn( "assert.push is deprecated and will be removed in QUnit 3.0." +
-			" Please use assert.pushResult instead (http://api.qunitjs.com/pushResult/)." );
+		if ( console ) {
```

Comment:
We should add a comment in the line above `// Supports IE9, not throwing if console is not active. Ref gh-1092`